article/abstract.tex

```diff
@@ -1,13 +1,13 @@
 \section{Abstract}
 
 The value of experimental research articles is inextricably contingent on data analysis results which substantiate their claims.
-However, the intricacy of data analysis procedures and their high reliance on extrinsic tools makes the computation component notoriously fragile.[CITATION NEEDED]
-The inability to re-use procedures due to instability thus endangers their value as a repository of procedural knowledge.
+However, it is well-established that the intricacy of data analysis procedures and their high reliance on extrinsic tools makes the computation component fragile.
+The inability to re-use code due to instability thus limits the lasting value of the work as a repository of procedural knowledge.
 
-It is therefore of crucial importance to approach a higher standard of what constitutes "complete analysis instructions"
+It is therefore of crucial importance to approach a higher standard of what constitutes "complete analysis instructions" to demonstrate the continued reexecutablity and portability of research, especially "keystone results".
 
-In this study, we examine a peer-reviewed neuroimaging experiment that uses automated data analysis instructions and extend it using a collection of best practices, including YODA principals for data management, containers for preservation, Gentoo for long term flexibility.
-We have automated the execution pipeline from end-to-end, dynamically retrieving raw data, performing the analysis, dynamically generating the statistics in the text, figures, and rendering the final article file. 
+In this study, we examine a peer-reviewed neuroimaging experiment that uses automated data analysis instructions and we extend it using a collection of best practices, including YODA principals for data management, containers for preservation, Gentoo for long term flexibility.
```

Comment:
```suggestion
In this study, we examine a peer-reviewed neuroimaging experiment that uses automated data analysis instructions and extend it using a collection of best practices, including containers for encapsulation of execution environments, YODA principles for composition, and DataLad for data management and preservation.
```

Reply:
what about Gentoo?

Reply:
Nobody besides Gentoo folks would consider it to be "best practices".  Moreover Gentoo was not taken out of the blue -- it was used in the original paper.

Reply:
I see what you mean, but the claim I'm trying to make is that because we use Gentoo we have an additional layer of protection since Gentoo should make it theoretically easier to rebuild the container and perhaps bump dependency versions. 

Since the original didn't use containers, "rebuildable containers" is a new improvement toward reproducibility, and thought it at least deserved a mention.

I concede it's not the most important point, so I'm ok to remove, even if I am still attached to mentioning "rebuildable containers"